.gitignore

```diff
@@ -1,4 +1,5 @@
 .DS_Store
+.vscode/
```

Comment:
Not sure if we should explicitly add this here.. Ideally, every dev should kind of add it anyways.
@rrtheonlyone should we add vsc gitignore in this PR? I see that IDEA is added but not vsc.

Reply:
I think for now, you could try to add this line to your own global `.gitignore` first instead of project local.

Reply:
Got it. I will get to it now.